lib/authentication/view/sign_up_view.dart

```diff
@@ -45,25 +51,43 @@ class _SignUpViewState extends State<SignUpView> {
     _fetchFilter();
   }
 
+  void parse(TextEditingController email, TextEditingController firstName,
+      TextEditingController lastName) {
+
+    String emailName = email.text.replaceAll(RegExp(r'[0-9]'), '');
+    List<String> name = emailName.split('.');
```

Comment:
Call it `names`. It's a list of strings representing names, after all, not a single name.